conda-recipe/meta.yaml

```diff
@@ -26,6 +26,7 @@ requirements:
       - qtconsole
       - qtpy
       - timechart
+      - numpydoc
```

Comment:
Fine with `numpydoc`. Looks like it requires hardly anything:

https://github.com/numpy/numpydoc/blob/master/.travis.yml